tracer/tracer.go

```diff
@@ -45,6 +49,18 @@ func WithSampler(sampler Sampler) Option {
 	}
 }
 
+func WithTraceIdHeader(traceIdHeader string) Option {
```

Comment:
Instead of these two options, let's just have:

```
func WithJaegerPropagator(tracerId, baggagePrefix string) Option {
        return func(args *WavefrontTracer) {
                 var options []JaegerOption
                 // append Jaeger Options as required if tracerId and baggagePrefix are non-empty
		args.jaegerWavefrontPropagator = NewJaegerPropagator(args, options)
	}
}
```